crates/ruff/src/rules/flake8_pyi/rules/simple_defaults.rs

```diff
@@ -399,9 +429,10 @@ pub(crate) fn argument_simple_defaults(checker: &mut Checker, arguments: &Argume
 
 /// PYI015
 pub(crate) fn assignment_default_in_stub(checker: &mut Checker, targets: &[Expr], value: &Expr) {
-    let [target] = targets else {
+    if targets.len() != 1 {
         return;
-    };
+    }
+    let target = &targets[0];
```

Comment:
What's the motivation for this change? I believe it's semantically equivalent, but the former enforces the condition with a single match.

Reply:
Whaaat, I don't remember changing that part of the code. I'll revert the changes.